### PR TITLE
[FIX] mail: Fix read more link navigation issue

### PR DESCRIPTION
**Pull request by jat-odoo**
Description of the issue/feature this PR addresses:
- Task: https://www.odoo.com/web#id=672040&view_type=form&model=project.issue&action=609&menu_id=4720

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/js/thread.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ var Thread = Widget.extend({
 
         this.$('[data-o-mail-quote="1"]').each(function () {
             var $content = $(this);
-            var $read_more = $('<a class="o_mail_read_more" href="#"></a>').text(read_more);
+            var $read_more = $('<a class="o_mail_read_more" />').text(read_more);
             var is_read_more = true;
             $read_more.click(function() {
                 is_read_more = !is_read_more;
```
Comment (aab-odoo) [inline addons/mail/static/src/js/thread.js]: This function takes event `e` as argument. `e.preventDefault()` would then do the trick.

Comment (jat-odoo) [inline addons/mail/static/src/js/thread.js]: @aab-odoo , Yes, It's working fine! Thanx :+1:

**Comment by aab-odoo** [comment]
Is there an associated Odoo issue?

**Comment by aab-odoo** [comment]
Merged in saas-10 at 6c40782afd9d2231d01c321c335bfce00d682ebd.
Thanks.

**Pull request closed without merging**